### PR TITLE
Add keynote speakers to UGM page

### PR DESCRIPTION
**Pull request by jennaswa**

### CODE REVIEW

**Review by richardjgowers — APPROVED** [review]

File: pages/ugm2024.md [hunk 1]
```diff
@@ -16,7 +16,23 @@ The UGM will bring together users of the MDAnalysis package from different commu
...
 ## Keynote Speakers
-Keynote speakers will be announced in Spring 2024.
+### Dr. Antonia Mey
+
+<img
+src="/public/images/UGM2024photo_Mey.jpg"
+title="Dr. Antonia Mey" alt="Dr. Antonia Mey"
+style="float: left; width: 220px; height: 220px; border-radius: 20px; border: 15px solid white" />
+
+Antonia has worked on Markov State models with Frank Noe and alchemical free energy calculations with Julien Michel before starting her own research group in 2020. Her group works on computational biophysics problems using machine learning and simulations to understand protein function. She is an advocate for open-source software and diamond open access publishing models. She has contributed to pyEmma, BioSimSpace and she and her group (https://mey-research.org) are avid users of MDAnalysis.
```
Comment (orbeckst) [inline pages/ugm2024.md:26]: I'd add where she has her position

... her own research group in 2020 at the [University of Edinburgh in the School of Chemistry](http://www.chem.ed.ac.uk/)

**Review by orbeckst — APPROVED** [review]
looking good, just one optional suggestion

File: pages/ugm2024.md (hunk 1 above)
Comment (orbeckst) [reply pages/ugm2024.md:26]: maybe not needed to link... I just copied and pasted

Comment (jennaswa) [reply pages/ugm2024.md]: Good suggestion, thanks! I've decided not to link as we haven't linked in other places.

**Review by RMeli — APPROVED** [review]

**Pull request merged**